user/developer.md

```diff
@@ -27,6 +27,22 @@ We've created an [API Explorer](https://developer.travis-ci.org/) that closely
 integrates with V3, updating automatically when new endpoints are added, and
 includes a useful tool for exploring endpoints.
 
+## API V2.1
+
+We've released an update to the Travis CI API V2, which is API V2.1. This update allows users of Travis CI for private projects to make certain build information available through the API.
+
+For users of Travis CI for public repositories there is no change.
+
+API V2.1 is identical to API V2 **except for the following breaking changes**:
+
+* For public repositories, unauthenticated requests receive an HTTP 200 or 400 error.
```

Comment:
According to Sven's docs here, the return status codes can be 200 or 404 https://gist.github.com/svenfuchs/5df1b12efd607d8914ceb7cd3dadad93

Reply:
Fixed